spec/lucky/cookies/flash_store_spec.cr

```diff
@@ -92,12 +92,14 @@ describe Lucky::FlashStore do
     it "returns true if there are key/value pairs" do
       flash_store = build_flash_store({"some_key" => "some_value"})
 
+      # ameba:disable Performance/AnyInsteadOfEmpty
       flash_store.any?.should be_true
```

Comment:
This suggested doing `!(flash_store.empty?).should`, but the spec is actually testing the `any?` method 😂